RecDP/pyrecdp/primitives/operations/text_pii_remove.py

```diff
@@ -48,6 +48,28 @@ def process(sample):
 class PIIRemoval(BaseLLMOperation):
     def __init__(self, text_key='text', inplace=True, model_root_path="", debug_mode=False,
                  entity_types: List[PIIEntityType] = None):
+        """
+        Args:
+            text_key(str): The name of the text field to be processed. Default is 'text'.
+            inplace(bool): Whether to update on the original text field. Default is True.
+            model_root_path(str): The root path to the model files.
+                For example, if you manually download the Hugging Face model `bigcode/starpii` for NER detection
+                from [Hugging Face](https://huggingface.co/bigcode/starpii)
+                to `/path/to/your_model_root_path/bigcode/starpii`,
+                then the `model_root_path` should be `"/path/to/your_model_root_path/"`.
+                Default is None, and it will auto download the huggingface model.
+            debug_mode(bool): Whether to enable debug mode. Default is False.
+            entity_types: The types of personal information to be removal. Default is None.
```

Comment:
please provide user a list of options for this parameter

Reply:
updated